dns_sd_avahi.c

```diff
@@ -60,6 +60,39 @@ void dnssd_free_discovery_data(struct dns_sd_discovery_data *d)
 	free(d);
 }
 
+static void __avahi_process_resolved(struct dns_sd_discovery_data *ddata, const AvahiAddress *addr,
```

Comment:
I suggest avoiding the use of double underscore at the beginning of name.

Reply:
Such naming seems to be reserved by gnu compilers: https://www.gnu.org/software/libc/manual/html_node/Reserved-Names.html

Reply:
Oh, didn't new this. Nice catch!